trackintel/analysis/location_identification.py

```diff
@@ -197,6 +197,8 @@ def freq_method(staypoints, *labels):
             group["duration"] = group["finished_at"] - group["started_at"]
         # pandas keeps inner order of groups
         sp.loc[sp["user_id"] == name, "purpose"] = _freq_transform(group, *labels)
+    if "purpose" not in sp.columns:  # if empty sp
+        sp["purpose"] = None
```

Comment:
Still related to the data type: shall we use `None`, `pd.NA` or `np.nan`? 

Edit: I think we need to use `pd.NA` for a consistent datatype of `Int64`.

Reply:
As the data type of "purpose" is `object` all three should work. But as `pd.NA` is intended for `Int64` and `np.nan` for `float`, I thought that `None` is good for `object`.